File: celsius_conversion/ruby_celsius_conversion/celsius_conversion.rb
#!/usr/bin/ruby -w
#comments

def toFahrenheit ( x )
  return  x * (9.0/5.0) + 32.0
end

def toCelsius ( x )
  return (x - 32.0) * (5.0/9.0)
end

def example_function(n)
  return n
end

def main()
  if ARGV.length != 0
    puts "Usage: #{$0} <noargs>"
    exit(1)
  end
  #for idx_i in (1...10)
  puts "toFahrenheit(#{21})=<#{toFahrenheit(21)}>"
  puts "toCelsius(#{70})=<#{toCelsius(70)}>"
  #end
end

main()

File: celsius_conversion/lua_celsius_conversion/celsius_conversion.lua
-- comments
function example_function (n)
  return n;
end

function toFahrenheit ( x )
  return  x * (9.0/5.0) + 32.0
end

function toCelsius ( x )
  return (x - 32.0) * (5.0/9.0)
end

function main()
  --for idx_i=1,14,1
  --do
  print("toFahrenheit(" .. 21 .. ")=<" .. tostring(toFahrenheit(21)) .. ">")
  print("toCelsius(" .. 70 .. ")=<" .. tostring(toCelsius(70)) .. ">")
  --end
end

main()
